setup.py

```diff
@@ -17,6 +17,7 @@
     "pluggy>=0.12,<1.0",
     "importlib-metadata>=0.12",
     "wcwidth",
+    "astor",
```

Comment:
Is this really needed, or can we work around it in someway? I would rather not having a new dependency unless strictly necessary.

Reply:
Basically this is needed to provide the hook users with the python code that contains the assertion, and astor is used to go from the ast to python source code.

I would also prefer not to add dependencies, but I am not sure if there is anything already in place that does that.

Reply:
@nicoddemus what about we incorporate the `codegen` module from `pytest-ast-back-to-python`? Or maybe the whole plugin?

Reply:
Not sure, it has really limited scope (only for developers), but perhaps in the future? 🤔

Reply:
Ok, so staying with `astor` as a dependency.